### PR TITLE
docs: add contributing documentation

### PR DESCRIPTION
**Pull request by andresz1**
## Contributing documentation

**TASK**: #160 

### Description, Motivation and Context
This PR add contributing documentation

### CODE REVIEW

File: documentation/contributing/GettingStarted.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import { Meta } from '@storybook/blocks'
+import { StoryHeading } from '@docs/helpers/StoryHeading'
+
+<Meta title="Contributing/Getting started" />
```
Comment (andresz1) [inline documentation/contributing/GettingStarted.mdx:4]: Maybe would be better a plain `CONTRIBUTING.md` file ?

**Comment by Powerplex** [comment]
To summarize, you suggest that everything related to the Spark workflow/guidelines is written in markdown, not "MDX for Storybook" so that it is accessible but does not appear  in storybook menu ?

**Comment by andresz1** [comment]
> To summarize, you suggest that everything related to the Spark workflow/guidelines is written in markdown, not "MDX for Storybook" so that it is accessible but does not appear in storybook menu ?

Exactly!

**Comment by Powerplex** [comment]
https://storybook.js.org/blog/structuring-your-storybook/

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**